src/new-bot/commands/expertise/update.js

```diff
@@ -36,16 +39,143 @@ function updateExpertise({userName, expertise, newValues}) {
   });
 }
 
-const updateMissingHandler = createMatcher({match: 'missing'}, (_, {user}) => {
-  return `> update missing for ${user.name} (coming soon)`;
-});
+function updateExpertiseDialog({
```

Comment:
This function has quite a lot of responsibilities. Is it possible to factor some of it out into functions that receive only the context they care about?